docs/flamegraphs.md

```diff
@@ -0,0 +1,9 @@
+# Flamegraphs
+
+To help in identifying performance bottlenecks and slow performing functions, Developer Tools includes a CPU time [flamegraph](http://www.brendangregg.com/flamegraphs.html) of the current page's execution. Open the Altis Developer Tools and navigate to the "Xray -> Flamegraph" panel.
+
+![](./qm-flamegraph.png)
+
+The flamegraph shows execution time along the X axis. The vertical axis displays the call-depth of the PHP stack at that point in time. The flamegraph can tell you very quickly what is using the most time in the current page generation. Items in the flamegraph as clickable to zoom-in on the specific call.
+
+Flamegraphs are generated with the [XHProf PHP extension](https://www.php.net/manual/en/book.xhprof.php), using sampling mode. The sampling interval is set to 5 milliseconds. This is considered the "resolution" of the performance profile, so you should not consider anything taking 5ms or faster to be accurately represented in the flamegraph.
```

Comment:
```suggestion
Flamegraphs are generated with the [XHProf PHP extension](https://www.php.net/manual/en/book.xhprof.php), using sampling mode. The sampling interval is set to 5 milliseconds. This is considered the "resolution" of the performance profile, so you should not consider anything taking 5ms or less to be accurately represented in the flamegraph.
```

Feels a bit clearer to me to make this small change. My developer brain is geared to think in operators like `<=` I reckon!

Reply:
I presume we're using Tideways' fork of XHProf for PHP7. I wonder if there's a better URL to link to instead of the php.net docs.